### ISSUE TITLE
Could not find the declaration file under "Node16" or higher

### ISSUE THREAD

**Issue by truemogician** [gregnb/filemanager-webpack-plugin#119]
## Issue

The `exports` field in `package.json` is kind of misconfigured, since it misses `types`, leading to the following error when setting `moduleResolution` to `Node16` or `NodeNext` in `tsconfig.json`:
```
error TS7016: Could not find a declaration file for module 'filemanager-webpack-plugin'
```
Here is a comment from TypeScript that explains this issue: [TypeScript module "Node16" does not resolve types of CJS module](https://github.com/microsoft/TypeScript/issues/49160#issuecomment-1137482639)  
Simply adding `types` in `exports` could address this issue.

## Your Environment

| Tech                       | Version |
| - | :-: |
| filemanager-plugin-webpack |   7.0.0    |
| node                       |  ^16.0.0   |

**Comment by sibiraj-s** [comment]
Fixed in #120

**Comment by sibiraj-s** [comment]
published in v8.0.0
